src/js/covidMap.js

```diff
@@ -441,6 +508,182 @@ fluid.defaults("fluid.covidMap.map", {
     }
 });
 
+// The context awareness handler for the desktop view
+fluid.defaults("fluid.covidMap.desktopViewHandler", {
+    gradeNames: "fluid.modelComponent",
+    model: {
+        // Initial visibility values for the desktop view
+        filterPanelVisible: true,
+        filterPanelExpanded: true,
+        resultsPageVisible: true,
+        resultsPageExpanded: true,
+        citiesListVisible: true,
+        citiesListExpanded: true,
+        hospitalPanelVisible: false,
+        hospitalPanelExpanded: true
+    }
+});
+
+// The context awareness handler for the mobile view
+fluid.defaults("fluid.covidMap.mobileViewHandler", {
+    gradeNames: "fluid.viewComponent",
+    selectors: {
+        filterCountOnMobile: ".fl-mapviz-filter-count-on-mobile",
+        locationButtonOnMobile: ".fl-mapviz-locations-button-on-mobile",
+        filterButtonOnMobile: ".fl-mapviz-filter-button-on-mobile"
+    },
+    components: {
+        filterCountOnMobile: {
+            type: "fluid.covidMap.filterCount",
+            container: "{that}.dom.filterCountOnMobile"
+        },
+        locationButtonOnMobile: {
+            type: "fluid.button",
+            container: "{that}.dom.locationButtonOnMobile",
+            options: {
+                modelListeners: {
+                    "showCitiesList": {
+                        path: "activate",
+                        changePath: "{map}.model.citiesListVisible",
+                        value: true,
+                        excludeSource: ["init"]
```

Comment:
Lots of excludeSource: ["init"] annotations in new-style code are a code smell that the interaction hasn't been encoded in a properly integral way and should aim to be removed